pkgs/by-name/mi/microfetch/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
```

```suggestion
}:

```
Not an issue but we usually put a new line here